### PR TITLE
Add saj component to docs

### PR DESCRIPTION
**Pull request by fredericvl**
**Description:**
The `saj` sensor will poll a [SAJ](https://www.saj-electric.com/) solar inverter and present the values as sensors in Home Assistant.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26902

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/saj.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: "SAJ Solar Inverter"
+description: "Instructions on how to connect your SAJ Solar Inverter to Home Assistant."
+ha_category:
+  - Energy
+logo: saj.png
+ha_iot_class: Local Polling
+ha_release: "0.100"
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/saj.markdown]: Please remove the `redirect_from`

File: source/_components/saj.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+## Configuration
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: saj
+    host: IP_ADDRESS_OF_DEVICE
+    monitored_conditions:
```
Comment (klaasnicolaas) [inline source/_components/saj.markdown]: Only required variables in the first example

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/saj.markdown (hunk 1 above)
Comment (fredericvl) [reply source/_components/saj.markdown]: Done

File: source/_components/saj.markdown (hunk 2 above)
Comment (fredericvl) [reply source/_components/saj.markdown]: Changed

**Review by MartinHjelmare — APPROVED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Thnx for the PR! 🥇

**Pull request merged**